test/boost/cdc_test.cc

```diff
@@ -849,3 +849,23 @@ SEASTAR_THREAD_TEST_CASE(test_row_delete) {
         BOOST_REQUIRE_EQUAL(*results[2].front(), data_value(cdc::operation::row_delete).serialize_nonnull());
     }, mk_cdc_test_config()).get();
 }
+
+SEASTAR_THREAD_TEST_CASE(test_mutation_without_timestamp) {
+    do_with_cql_env_thread([](cql_test_env& e) {
+        auto test = [&] (bool preimage) {
+            cquery_nofail(e, format("CREATE TABLE ks.tbl (pk int, ck int, val int, PRIMARY KEY(pk, ck)) WITH cdc = {{'enabled':true{}}}", preimage ? ",'preimage': true" : ""));
```

Comment:
BTW, libfmt prints booleans as "true"/"false". It's no big deal, so no need to change this line, but I think it's one little thing worth remembering.

Reply:
Added single quotes.